applications/luci-app-watchcat/luasrc/model/cbi/watchcat/watchcat.lua

```diff
@@ -1,53 +1,102 @@
 -- Copyright 2012 Christian Gagneraud <chris@techworks.ie>
+-- Copyright 2020 Nicholas Smith <nicholas@nbembedded.com>
 -- Licensed to the public under the Apache License 2.0.
+local net = require"luci.model.network".init()
+local sys = require "luci.sys"
+local ifaces = sys.net:devices()
 
-m = Map("system", 
-	translate("Watchcat"), 
-	translate("Watchcat allows configuring a periodic reboot when the " ..
-		  "Internet connection has been lost for a certain period of time."
-		 ))
+m = Map("system", translate("Watchcat"),
+		translate(
+			"Here you can set up several checks and actions to take in the event that a host becomes unreachable. " ..
+				"Click the <b>Add</b> button at the bottom to set up more than one action."))
 
-s = m:section(TypedSection, "watchcat")
+s = m:section(TypedSection, "watchcat", translate("Watchcat"), translate("These rules will govern how this device reacts to network events."))
 s.anonymous = true
 s.addremove = true
 
-mode = s:option(ListValue, "mode",
-		translate("Operating mode"))
-mode.default = "allways"
-mode:value("ping", "Reboot on internet connection lost")
-mode:value("allways", "Periodic reboot")
-
-forcedelay = s:option(Value, "forcedelay",
-		      translate("Forced reboot delay"),
-		      translate("When rebooting the system, the watchcat will trigger a soft reboot. " ..
-				"Entering a non zero value here will trigger a delayed hard reboot " ..
-				"if the soft reboot fails. Enter a number of seconds to enable, " ..
-				"use 0 to disable"))
-forcedelay.datatype = "uinteger"
-forcedelay.default = "0"
-
-period = s:option(Value, "period", 
-		  translate("Period"),
-		  translate("In periodic mode, it defines the reboot period. " ..
-			    "In internet mode, it defines the longest period of " .. 
-			    "time without internet access before a reboot is engaged." ..
-			    "Default unit is seconds, you can use the " ..
-			    "suffix 'm' for minutes, 'h' for hours or 'd' " ..
-			    "for days"))
-
-pinghost = s:option(Value, "pinghosts", 
-		    translate("Ping host"),
-		    translate("Host address to ping"))
+mode = s:option(ListValue, "mode", translate("Mode"))
+mode.default = "ping_reboot"
+mode:value("ping_reboot", translate("Ping Reboot: Reboot this device if a host is unreachable"))
```

Comment:
The TropDown gets quite long due to the translation. Can we not shorten this.

Reply:
Thanks for the feedback, will do. It just looks this way at the moment because of the auto formatter I have in my editor, which I need to fixup.

Reply:
> The TropDown gets quite long due to the translation. Can we not shorten this.

Yes, I'll rephrase and try to keep the option descriptions from getting too long by putting all of the context in the section's description.